cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${projectdir} build.vars)
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+
+helm_tag="$(cat ${projectdir}/_output/version)"
+CROSSPLANE_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}:${helm_tag}"
+K8S_CLUSTER="${BUILD_REGISTRY}-INTTESTS"
```

Comment:
Do we need to worry about isolation between multiple test runs? Or is it already taken care of?

Reply:
The names of k8s cluster and the produced  crossplane image are unique for each pr, so it should be isolated